meshtastic/interfaces/ble/compatibility_service.py

```
@@ -23,6 +23,49 @@
+            publishing_thread=self._publishing_thread_provider(),
+        )
+
+    def publish_connection_status(self, *, connected: bool) -> None:
+        """Publish connection status via the bound interface publishing thread."""
+        BLECompatibilityEventService.publish_connection_status(
+            self._iface,
+            connected=connected,
+            publishing_thread=self._publishing_thread_provider(),
+        )
+
+    def publish_connection_status_legacy(self, connected: bool) -> None:
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)